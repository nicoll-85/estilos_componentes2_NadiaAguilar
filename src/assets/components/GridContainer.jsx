import React from "react";
import styled from "styled-components";

const GridContainer = styled.div`
  display: grid;
  grid-template-columns: repeat(5, 1fr);
  gap: 10px;
`;

export default GridContainer;
