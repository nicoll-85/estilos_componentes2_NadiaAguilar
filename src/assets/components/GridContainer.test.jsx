// sum.test.js
import { expect, it } from "vitest";
import GridContainer from "./GridContainer";
import renderer from "react-test-renderer";

it("GridContainerTest", () => {
  const tree = renderer.create(<GridContainer />).toJSON();
  expect(tree).toMatchSnapshot();
});
