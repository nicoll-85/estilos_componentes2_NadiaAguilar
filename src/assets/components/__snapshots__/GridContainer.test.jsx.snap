// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`GridContainerTest 1`] = `
<div
  className="sc-beySPh hnlHBD"
/>
`;
